### PR TITLE
Added a landing image and a few modifications

### PR DESCRIPTION
**Pull request by frankmorara**
Hey @bitbadgers/scott-netbot check out my change.

### CODE REVIEW

**Comment by frankmorara** [comment]
Comments?

File: html_css_learning_models/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (kiambati-maxwel) [inline html_css_learning_models/.vscode/settings.json:2]: what is this part was it injected by live server...

**Review by kiambati-maxwel — COMMENTED** [review]
live

**Comment by kiambati-maxwel** [comment]
i finally resolved the merge conflict

git pull origin THE-NAME-OF-THE-BRANCH
git add -A
git commit -a  -m "--- message ---"

**Pull request merged**